src/pages/2021-04-27---hyperlocal-ice-cream-business/index.md

```diff
@@ -0,0 +1,52 @@
+
+## Story/Idea
+
+We had our first child last August (pandemic baby, baby). As a treat, I fulfilled a childhood dream and bought a fancy ice cream maker. Which ended up being great fun as the fresh ice cream helped distract / soothe our stressed / exhausted new parent brains. For a week or so post-birth I was making a new flavor once or twice a day for ourselves and my in-laws who were staying helping out.
+
+![One of the first ice creams we made](./first-ice-cream.JPG)
+_One of the first ice creams we made last August_
+
+One of the problems with making ice cream (all desserts really) is that once you make it you... well have to eat it. Ice cream, while delicious, is not perhaps the key component of a healthy balanced diet.
+
+So I have the dilemma that I want to make all the ice cream flavors but can't eat them fast enough.
+
+My wife and I have also felt isolated during the pandemic (like most people) and want ways to (re)connect with neighbors and make friends.
+
+I've also been looking for good streaming projects for Gatsby to show off all the work we're doing on the framework and in [Gatsby Cloud](https://www.gatsbyjs.com/products/cloud/) like [functions](https://github.com/gatsbyjs/gatsby/discussions/30735) and [hosting](https://www.gatsbyjs.com/products/cloud/hosting/).
+
+So sprung forth the idea of creating a new ice cream micro-business. We create a web app where people could browse our flavors (seasonal from our backyard fruit trees and local grocery store) and schedule a time for us to deliver the home-make hyper-fresh, hyper-local, hyper-yummy ice cream.
+
+And then once a week or so, we'll strap the little guy into his stroller, meander over to the nearby grocery store to pick up ingredients, make the ice cream, then deliver the ice cream (leaving a bit left-over at home for tasting).
+
+We'll restrict stroller delivery to within a 1/2 mile (people further out can pick up)
+
+![little cutie](./little-cutie.jpg)
+_and who wouldn't want delicious fresh-made ice cream delivered by this cutie??_
+
+## Situated Software
+
+Something I've been thinking about for a decade+ is the idea of _Situated Software_. [Clay Shirky defines it as](https://www.gwern.net/docs/technology/2004-03-30-shirky-situatedsoftware.html):
+
+> software designed in and for a particular social situation or context. This way of making software is in contrast with what I'll call the Web School (the paradigm I learned to program in), where scalability, generality, and completeness were the key virtues.)
+
+Software is great. I've written, write, and will write a lot of it and interact with 10,000x more of it in all the physical/virtual parts of my life.
+
+But I think we should and will have a lot more of it.
```

Comment:
You already talked about the future, so why is this projection different than the previous sentence?